src/matcha_ml/cli/analytics.py

```diff
@@ -0,0 +1,26 @@
+"""Matcha CLI for opting out/in of collecting analytics."""
+import typer
+from rich import print
+
+from matcha_ml.services.global_parameters_service import GlobalParameters
+
+# create a typer app to group all analytics subcommands
+app = typer.Typer()
+
+
+@app.command()
+def opt_out() -> None:
+    """Disable the collection of anonymous usage data."""
+    print(
+        "Data collection has been turned off and no data will be collected - you can turn this back on by running the command: 'matcha analytics opt-in'"
+    )
+    return GlobalParameters().update_analytics_state(disable=True)
```

Comment:
Now that I see how it is implemented, is it worth just directly accessing the class property instead of having a method like this?
e.g.
```
GlobalParameters().analytics_opt_out = True
```
And we can add a setter to `analytics_opt_out`:

```python
@analytics_opt_out.setter
def analytics_opt_out(self, value: bool) -> None:
        self._analytics_opt_out = value
        self._update_global_config()
```

Reply:
Yeh, this makes sense. 

I think we'll ultimately refactor this such that there's a function in `core` which actually calls the setter rather than at the CLI level